tools/interop_matrix/client_matrix.py

```diff
@@ -200,7 +200,15 @@ def should_build_docker_interop_image_from_release_tag(lang):
         {
             'v1.6.6': None
```

Comment:
Can you comment out the previous version for now so they won't fail? I'll add them back in my PR.

Reply:
done